tests/parser/file_parser.cpp

```diff
@@ -31,6 +31,8 @@ TEST(Parser, parseSineModelFromFile)
     p.parseModel(fileContents("sine_approximations.xml"));
 
     EXPECT_EQ(size_t(0), p.errorCount());
+
+    printErrors(p);
```

Comment:
What's the point of printing the errors if we don't check what is being printed?